conf/templates/.gitignore

```diff
@@ -0,0 +1,5 @@
+*
```

Comment:
It would be better if this had a single, specific sub-directory whose contents were entirely ignored, rather than ignore files in this project directory.

Example: `conf/templates/user`, create a `conf/templates/user/.gitignore` file containing only `*` entry.